app/components/datepicker/DatepickerDirective.js

```diff
@@ -135,21 +135,9 @@
             }
         }
 
-        function showInfo(){
-            var modalInstance = $uibModal.open({
-                animation: true,
-                templateUrl: 'infoPopup.html',
-                controller: 'InfoWindowController',
-                size: 'lg',
-                resolve: {
-                    infoMsg: function(){
-                        return solrHeatmapApp.instructions.datepicker.instruction;
-                    },
-                    toolName: function(){
-                        return solrHeatmapApp.instructions.datepicker.toolTitle;
-                    }
-                }
-            });
+        function showDatepickerInfo() {
+            InfoService.showInfoPopup(solrHeatmapApp.instructions.datepicker.instruction,
```

Comment:
very good. like it, I would even go further and just send the key where the instructions would be found.
e.g.: `datepicker` or `export`